client/lib/plans/features-list.tsx

```diff
@@ -302,6 +304,8 @@ import ExternalLinkWithTracking from 'calypso/components/external-link/with-trac
 import MaterialIcon from 'calypso/components/material-icon';
 import { DOMAIN_PRICING_AND_AVAILABLE_TLDS } from 'calypso/lib/url/support';
 
+const locale = useLocale();
```

Comment:
Cannot be here, need to look up what's non-component way of getting user locale.

Reply:
Or just remove `englishLocales.includes( locale )` check entirely, not a biggie.